readme.md

```diff
@@ -35,14 +52,18 @@ Type: `string`
 
 String with CSS.
 
+
 ## options
 
-### all
+### preserve
 
-Type: `boolean`  
-Default: `false`
+Type: `boolean`, `RegExp`, or `function` 
+Default: `true`
 
-Whether *important* CSS comments *(those starting with `/*!`)* should be stripped.
+- `preserve: true` &mdash; (default) preserve comments that use the `/*! */` syntax;
```

Comment:
Replaced. Out of curiosity, what's the reason? I was taught that an em-dash is more typographically/grammatically correct in this case.